schema/stage_ci.yaml

```diff
@@ -1,4 +1,7 @@
 properties:
+  buildIntelligence:
```

Comment:
We use kebab case as opposed to camel case, favoring terse naming where possible. Also I think build intelligence is enabled by default (or should be) in which case we would use `disabled: true` as opposed to `enabled`(more details [here](https://harness.atlassian.net/wiki/spaces/~5f1f0648e8c4560022265233/pages/21100134856/RFC-005+-+Yaml+Syntax+Simplification#Booleans))

Reply:
Thanks for the feedback, for now I am able to handle this in the downgrader logic so I no longer need the schema change, sorry for the distraction.